armi/reactor/reactorParameters.py

```diff
@@ -723,42 +770,47 @@ def defineCoreParameters():
 
         pb.defParam(
             "breedingRatio",
-            units="N/A",
+            units=units.UNITLESS,
             description="Breeding ratio of the reactor",
             default=0.0,
         )
 
-        pb.defParam("ConvRatioCore", units="?", description="?")
+        pb.defParam(
+            "ConvRatioCore",
+            units=units.UNITLESS,
+            description="Conversion ratio of the core",
+        )
 
-        pb.defParam("absPerFisCore", units="?", description="?")
+        pb.defParam(
+            "absPerFisCore",
+            units=units.UNITLESS,
+            description="absorptions per fission in core",
+        )
 
         pb.defParam(
             "axialExpansionPercent",
-            units="%",
+            units=units.PERCENT,
             description="Percent of axial growth of fuel blocks",
             default=0.0,
         )
 
-        pb.defParam("coupledIteration", units="?", description="?", default=0)
-
-        pb.defParam("fisFrac", units="?", description="?")
-
-        pb.defParam("fisRateCore", units="?", description="?")
+        pb.defParam(
+            "coupledIteration",
+            units=units.UNITLESS,
+            description="Pre-defined number of tightly coupled iterations.",
+            default=0,
+        )
 
         pb.defParam(
-            "maxdilationTotal",
-            units="?",
-            description="?",
-            default=0.0,
-            categories=["block-max"],
+            "fisFrac",
+            units=units.UNITLESS,
+            description="Percent of fissions in fertile nuclides",
         )
 
         pb.defParam(
-            "maxresidence",
-            units="?",
-            description="?",
-            default=0.0,
-            categories=["block-max"],
+            "fisRateCore",
+            units="fissions/s",
+            description="peak/average fission rate in core?",
```

Comment:
```suggestion
            units=units.UNITLESS,
            description="peak/average fission rate in core",
```

Reply:
lol

Chris just had me change this from UNITLESS to "fissions/s" like 30 minutes ago.

Thoughts?

Reply:
(I obviously agree with removing the `?` though.)

Reply:
I know, Chris assumed what the parameter was based on the name, which is not entirely unreasonable. Unfortunately, this parameter has a terrible name. I commented on that thread: https://github.com/terrapower/armi/pull/1345#discussion_r1265914298